QE-1

snegok369 opened the pull request:
Решить квадратное уравнение ax^2+bx+c=-7. На вход в программу подаются значения a,b,c. На выходе хотим получить решение уравнения ( в процессе )

Commit:
QE-1
Решить квадратное уравнение ax^2+bx+c=-7. На вход в программу подаются значения a,b,c. На выходе хотим получить решение уравнения ( в процессе )

Commit:
QE-1
Решить квадратное уравнение ax^2+bx+c=-7. На вход в программу подаются значения a,b,c. На выходе хотим получить решение уравнения ( в процессе )

ione88 commented on Krujka/src/main/java/com.company/Krujka.java:
```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
         int b = 2;
         double c = 2.75;
-        double d = b * 2 - 4 * a * c;
+        int d = (int) (b * 2 - 4 * a * c);
```
Зачем это? норм же с double работало?

snegok369 replied:
так ответ -7
а с дабл был ответ -7.0

ione88 replied:
не пугай меня))
ты же понимаешь, что -7 и -7.0 это одинаковые числа.
а если бы ты выбрал другие коэффициенты? и число не было б целым.
тогда ответы были б разные.

тут разница только в представление этого числа double лучше подходит для решения квадратного уровления. я б наоборот все числа double  сделал

snegok369 replied:
я поменял

Commit:
QE-1
Решить квадратное уравнение ax^2+bx+c=-7. На вход в программу подаются значения a,b,c. На выходе хотим получить решение уравнения ( в процессе )

Commit:
ты дыщ
